lib/shared/helpers.js

```diff
@@ -57,8 +57,9 @@ module.exports = function (jade, getTemplate) {
         },
 
         json: function (object, spacing) {
-            return JSON.stringify(object, null, spacing);
+            var regex = /[\u0000\u00ad\u0600-\u0604\u070f\u17b4\u17b5\u200c-\u200f\u2028-\u202f\u2060-\u206f\ufeff\ufff0-\uffff]/g
```

Comment:
I suggest moving the pattern created outside of the function scope making it a `const`.